ui/apps/platform/src/hooks/useURLSort.test.js

```diff
@@ -149,7 +149,7 @@ describe('useURLSort', () => {
             expect(result.current.getSortParams('Status').columnIndex).toEqual(1);
             expect(result.current.getSortParams('Status').sortBy.index).toEqual(0);
             expect(result.current.getSortParams('Status').sortBy.direction).toEqual('desc');
-            expect(result.current.getSortParams('Bogus').columnIndex).toEqual(undefined);
+            expect(result.current.getSortParams('Bogus').columnIndex).toEqual(-1);
```

Comment:
PatternFly declares the `columnIndex` value here to be a number, but the previous `useURLSort()` was incorrectly setting this to `undefined` when a matching column was not detected.

Although this is a change in behavior, however the `columnIndex` is only ever used to feed back as an argument to the `onSort` callback of the `<Th>` component. There is a single case where we rely on this value in `apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ScanConfigClustersTable.tsx#L60`, and both `-1` and `undefined` will have the same effect.